src/main/java/com/autotune/analyzer/services/BulkService.java

```diff
@@ -0,0 +1,126 @@
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *******************************************************************************/
+package com.autotune.analyzer.services;
+
+import com.autotune.analyzer.serviceObjects.BulkInput;
+import com.autotune.analyzer.serviceObjects.BulkJobStatus;
+import com.autotune.analyzer.workerimpl.BulkJobManager;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Map;
+import java.util.UUID;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+
+/**
+ *
+ */
+@WebServlet(asyncSupported = true)
+public class BulkService extends HttpServlet {
+    private static final long serialVersionUID = 1L;
+    private static final Logger LOGGER = LoggerFactory.getLogger(BulkService.class);
+    private ExecutorService executorService = Executors.newFixedThreadPool(10);
+    private Map<String, BulkJobStatus> jobStatusMap = new ConcurrentHashMap<>();
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        super.init(config);
+    }
+
+    /**
+     * @param req
+     * @param resp
+     * @throws ServletException
+     * @throws IOException
+     */
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String jobID = req.getParameter("jobID");
+        BulkJobStatus jobStatus = jobStatusMap.get(jobID);
+
+        if (jobStatus == null) {
+            resp.setStatus(HttpServletResponse.SC_NOT_FOUND);
+            resp.getWriter().write("{\"error\":\"Job not found\"}");
+        } else {
+            try {
+                resp.setContentType("application/json");
+                resp.setCharacterEncoding("UTF-8");
+                // Return the JSON representation of the JobStatus object
+                ObjectMapper objectMapper = new ObjectMapper();
+                String jsonResponse = objectMapper.writeValueAsString(jobStatus);
+                resp.getWriter().write(jsonResponse);
+            } catch (Exception e) {
+                e.printStackTrace();
+            }
+        }
+    }
```

Comment:
Dont we use a generic template for GET APIs?
Also please use constants for all of the strings

Reply:
done